### PR TITLE
[1921] Fix lint issues caused by landing embed.js

### PR DESCRIPTION
**Pull request by humphd**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e1ac1182aa977f1/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/e1ac1182aa977f1/output.txt

Total script time: 0.74 mins

File: src/embed.js [hunk 1]
```diff
@@ -6,18 +6,28 @@ function init( window, document ) {
 
   function requestFullscreen( elem ) {
     // Prefix + case differences.
-    elem.requestFullscreen && elem.requestFullscreen();
-    elem.mozRequestFullscreen && elem.mozRequestFullscreen();
-    elem.mozRequestFullScreen && elem.mozRequestFullScreen();
-    elem.webkitRequestFullscreen && elem.webkitRequestFullscreen();
+    if ( elem.requestFullscreen ) {
```
Comment (secretrobotron) [inline src/embed.js:9]: Paul Irish did this sort of thing really elegantly with rAF here: http://paulirish.com/2011/requestanimationframe-for-smart-animating/

You _might_ care about making it look like that, but not necessary.

**Comment by jbuck** [comment]
Rebased on master: https://github.com/mozilla/butter/commit/e799162f8eb8527ceddb68d1e98db089c7fe429e

**Pull request closed without merging**